src/ext/CGAL/OGL_VBO_helper.h

```diff
@@ -0,0 +1,373 @@
+// Copyright (c) 1997-2002  Max-Planck-Institute Saarbruecken (Germany).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org); you may redistribute it under
+// the terms of the Q Public License version 1.0.
+// See the file LICENSE.QPL distributed with CGAL.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL: svn+ssh://scm.gforge.inria.fr/svn/cgal/trunk/Nef_3/include/CGAL/Nef_3/OGL_helper.h $
```

Comment:
Can we base that on the latest version of the file? Otherwise it might be an issue license wise.

Reply:
This file was a copy/paste from OGL_helper.h, then changes were made and duplicate code was deleted. Not sure what level of change is considered enough to change the license?

Reply:
Oh, indeed, looks like that's quite a legacy issue:

Do you think it's possible to update both files to new versions? Something like:
- Removing both existing files
- Applying the changes to the latest CGAL distributed header
- Add the new files (e.g. with _5 suffix to indicate we are basing this on the CGAL 5 header)

I can't really judge how much effort that is yet, but it would great to find a solution for this.

Reply:
I moved OGL_VBO_helper.h to CGAL_OGL_VBO_helper.h and changed the license to OpenSCAD. This should be acceptable since it is overriding the original class, and the code used for VBO implementation is substantially different. As for the original OGL_helper.h, there are a few differences wiht the CGAL 5.1.2 version of the file.

It would be better if that work was done under a different PR specific to cleaning up the license since it affects the current baseline, not just the VBO implementation.

Reply:
Thanks, sounds good.